### PR TITLE
add Madagascar_RTC_autorunonce

### PR DESCRIPTION
**Pull request by xyb994**

### CODE REVIEW

File: examples/Madagascar_RTC_autorunonce [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+sudo apt-get -y remove fake-hwclock
+sudo update-rc.d -f fake-hwclock remove
+
+# second autorunonce
+mv /boot/autoranonce* /boot/autoranonce.first
+sync
+cat <<EOF > /boot/autorunonce
+#!/bin/sh
+sleep 10
+while ! curl -X GET https://google.com ; do
```
Comment (i5o) [inline examples/Madagascar_RTC_autorunonce]: kinda scared about this one,
if the user doesn't have an ehternet/wifi connected it will never boot?

Comment (dogi) [reply examples/Madagascar_RTC_autorunonce]: I know but so far have no better solution ...
should we raise that working upstream ethernet is required

Comment (i5o) [reply examples/Madagascar_RTC_autorunonce]: a weird solution would be to put this on a cron job (or find a way to run this when an internet connection is found if-up.d maybe?).

once the cronjob is done (internet connection found) remove the entry -crontab- from the system

Comment (dogi) [reply examples/Madagascar_RTC_autorunonce]: hmm maybe a task for an other issue ;)

**Pull request merged**